contrib/endpoints/src/api_manager/service_control/aggregated.cc

```diff
@@ -401,15 +396,11 @@ void Aggregated::Quota(const QuotaRequestInfo& info,
     delete response;
   };
 
-  AllocateQuotaRequest* quota_request_copy = new AllocateQuotaRequest(*request);
-
   // TODO(jaebong) Temporarily call Chemist directly instead of using service
   // control client library
   Call(*request, response,
-       [this, quota_request_copy, response,
-        check_on_done](::google::protobuf::util::Status status) {
-         delete quota_request_copy;
-         check_on_done(status);
+       [quota_on_done](::google::protobuf::util::Status status) {
```

Comment:
just pass in quota_on_done, 

Call(*request, response, quota_on_done, trace_span.get());